lib/views/user/_signin.html.erb

```diff
@@ -1,39 +0,0 @@
-<div id="signin">
-
-<%= form_tag({:action => "signin"}, {:id => "signin_form"}) do %>
-    <%= foi_error_messages_for :user_signin %>
-
-    <!--<% if not sign_in_as_existing_user %>
-    <h2><%= _('If you\'ve used {{site_name}} before', :site_name=>site_name)%></h2>
-    <% end %>-->
-
-    <p>
-    <label class="form_label" for="user_signin_email"><%= _('Your e-mail:')%></label>
-    <%= text_field 'user_signin', 'email', { :size => 20, :tabindex => 120 } %>
```

Comment:
Looks different to me:

```diff
- <%= text_field 'user_signin', 'email', { :size => 20, :tabindex => 120 } %>
+ <%= text_field 'user_signin', 'email', { :size => 20, :tabindex => 60 } %>
```

https://github.com/mysociety/alaveteli/blob/develop/app/views/user/_signin.html.erb#L7

See https://github.com/mysociety/derechoapreguntar-theme/commit/458ad320a0b9dd9bdf3d90cdc0bb05529b519606#diff-0c4b67a5a78077d4a261901b2022850aR11